tests/integration_test/integration_test.sh

```diff
@@ -481,10 +485,13 @@ docker run -i --rm \
               -e SS2_WORKFLOW_NAME="${SS2_WORKFLOW_NAME}" \
               -e SS2_VERSION="${SS2_VERSION}" \
               -v "${VAULT_TOKEN_PATH}":/root/.vault-token \
-              -v "${PWD}/lira/deploy/config_files":/working \
+              -v "${PWD}":/working \
+              --privileged \
```

Comment:
curious question: what is this for?

Reply:
The --privileged flag allows you to access files in a volume which has been mapped to the docker container. This was being problematic prior to the use of this flag, I think due to a docker upgrade